proxy_headers.go

```diff
@@ -51,9 +51,10 @@ func ProxyHeaders(h http.Handler) http.Handler {
 		if scheme := getScheme(r); scheme != "" {
 			r.URL.Scheme = scheme
 		}
-		// Set the host with the value passed by the proxy
-		if r.Header.Get(xForwardedHost) != "" {
-			r.Host = r.Header.Get(xForwardedHost)
+		// Set the host with the value passed by the first proxy
+		if forwardedHost := r.Header.Get(xForwardedHost); forwardedHost != "" {
+			forwardedHostSlice := strings.Split(forwardedHost, ", ")
+			r.Host = forwardedHostSlice[0]
```

Comment:
perhaps you can consider doing it the same way as propossed in https://gitlab.com/gitlab-org/gitlab-pages/-/merge_requests/386#note_536271745

```suggestion
			forwardedHostSlice := strings.Split(forwardedHost, ",")
                        r.Host = strings.TrimSpace(forwardedHostSlice[0])
```

Reply:
Thanks @jaimem88, done.